android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/TransactionParams.kt

```diff
@@ -0,0 +1,25 @@
+package com.solana.mobilewalletadapter.clientlib
+
+import com.solana.mobilewalletadapter.common.ProtocolContract
+import com.solana.mobilewalletadapter.common.protocol.CommitmentLevel
+
+sealed class RpcCluster(val name: String) {
+    object MainnetBeta : RpcCluster(ProtocolContract.CLUSTER_MAINNET_BETA)
+    object Testnet : RpcCluster(ProtocolContract.CLUSTER_TESTNET)
+    object Devnet : RpcCluster(ProtocolContract.CLUSTER_DEVNET)
+}
+
+open class TransactionParams(
+    val commitmentLevel: CommitmentLevel,
+    val cluster: RpcCluster,
+    val skipPreflight: Boolean,
+    val preflightCommitment: CommitmentLevel?
+)
+
+//TODO: We can add other defaults as they become relevant
+object DefaultTestnet : TransactionParams(
```

Comment:
Thought it might be nice to have some "default" options people could use.

Reply:
The protocol itself defines defaults. Perhaps the right answer here is to allow these to be nullable, and defer to protocol (or wallet) defaults if not specified?

Reply:
Well if you want to set custom values, you can just instantiate the `TransactionParams` class from line 12. Both `CommittmentLevel` values should be nullable though?